platform/golden/cfg/preseed-debian-11.cfg

```diff
@@ -69,6 +81,7 @@ d-i partman-partitioning/confirm_write_new_label boolean true
 d-i partman/choose_partition select finish
 d-i partman/confirm boolean true
 d-i partman/confirm_nooverwrite boolean true
+d-i partman-basicfilesystems/no_swap boolean false
```

Comment:
Do we need a swap partition?

Reply:
I have found that swap enablement in partitioning is not modifiable without a custom partition definition.